usaspending_api/api_contracts/contracts/v2/disaster/recipient/location/loans.md

```diff
@@ -37,15 +37,19 @@ Returns loan spending details of recipients' location receiving supplemental fun
                         "description": "Description text of 090, for humans",
                         "children": [],
                         "count": 54,
-                        "face_value_of_loan": 89.01
+                        "face_value_of_loan": 89.01,
```

Comment:
And based on [this mock](https://bahdigital.invisionapp.com/share/UCIAEBPVHMG#/296042273_COVID-_Recipients_-_Recipient_Location_-_State_-_Loans), I think we should have a `geo_layer` request param in this file.

Reply:
added